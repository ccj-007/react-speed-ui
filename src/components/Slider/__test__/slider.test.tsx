import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { Slider } from '../../index';

describe('test Slider component', () => {
  it('should render', () => {
    const { asFragment } = render(<Slider defaultVal={0} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
