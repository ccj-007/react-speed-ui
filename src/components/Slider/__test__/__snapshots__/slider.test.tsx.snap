// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Slider component should render 1`] = `
<DocumentFragment>
  <div
    class="speed-slider css-jxws1s-Slider"
  >
    <div
      class="speed-slider-content"
    >
      <div
        class="speed-slider-select css-13sp9s-Slider"
      />
      <div
        class="speed-slider-gan css-pw240g-Slider"
      />
    </div>
  </div>
</DocumentFragment>
`;
